src/Filament/Resources/FormResource.php

```diff
@@ -103,7 +103,7 @@ function (Get $get) {
                 Toggle::make('only_for_logged_in')->label('Restrict to logged in users'),
                 Repeater::make('notification_emails')
                         ->schema([
-                            TextInput::make('email')->email(),
+                            TextInput::make('email')->email()->required(),
```

Comment:
This doesn't make sense when the helper text says "Leave blank for no notifications".

I think let's remove the helper text.